tests/python/tests/http_server/test_headers_sent.py

```diff
@@ -0,0 +1,13 @@
+import socket
+from python.lib.testcase import KphpServerAutoTestCase
+from python.lib.http_client import RawResponse
+
+class TestHeadersSent(KphpServerAutoTestCase):
```

Comment:
Try to find in https://github.com/php/php-src/tree/PHP-7.4.33 tests which are relative for `headers_sent` and enable ones in https://github.com/VKCOM/kphp/blob/master/tests/zend-test-list

How to run Zend tests: https://vkcom.github.io/kphp/kphp-internals/developing-and-extending-kphp/writing-and-running-tests.html#3-running-zend-tests-partially

Reply:
found this one:
https://github.com/php/php-src/blob/aff044857f24042b93e38226e33228c43d01ef91/ext/standard/tests/general_functions/head.phpt#L6

but kphp won't compile var_dump(header(...));

do we have a flag to ignore void usage? or do I just copy that test into the repo and adjust to compile?

Reply:
Sadly. Yes, copy a test and adjust

Reply:
I propose to drop the idea of phpt test for this function, python test should be enough.

I tried to adjust the original test but it didn't work because:
phpt in php-src are ran using special cli tool and not just php, so that the text above <?php does not trigger echo, in php it does so there's headers_sent() is immediately true, I tried to use <?php from the get go like so:
```
<?php
$val = headers_sent();
?>
--TEST--
....
--FILE--
<?php
...

but than I have to use flush in order to achieve different values for headers_sent, but flush is NOT supported in kphp during CLI mode

on top of that the `header` function is highly in incompatible, it does not produce warnings that php does, and does produces warning where php DOES NOT, the expression `var_dump(header())` cannot be compiled, so basically I'll have to rewrite the entire test and even after that is still won't work because flush is not working in cli mode in kphp